dut/drivers.wake

```diff
@@ -0,0 +1,59 @@
+
+tuple SkeletalDriverPlan =
+  global DuhFile:             Path
+  global CompanyName:         String
+  global DeviceName:          String
+  global LocalMetalDirectory: String
+  global DUTPlan:             RocketChipDUTPlan
+
+global def makeSkeletalDriverPlan = SkeletalDriverPlan _ _ _ _ _
+
+def getDevicePlan deviceName =
+  def plans = subscribe skeletalDriverPlans
```

Comment:
Can you document the `skeletalDriverPlans` pub/sub topic? This is an important API for users of this flow, since they need to understand what data type should be published and what the expected behavior is of publishing to this topic.

Reply:
done